### PR TITLE
Changes based on Heidi's feedback

### PR DESCRIPTION
**Pull request by diberry**

### CODE REVIEW

**Comment by diberry** [comment]
@HeidiSteen This addresses your concerns - hopefully.

File: README.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ You can view additional samples for Javascript/Typescript in the [azure-sdk-for-
 Add search to a book catalog website. 
 
 This sample includes:
-* A React website for [Azure Cognitive Search](https://docs.microsoft.com/en-us/azure/search/search-what-is-azure-search): the client application provides the search user interface.
-* A Function app which calls Azure Function API: It leverages the [Azure SDK for Javascript/Typescript](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/search/search-documents/).
+* A client React website for [Azure Cognitive Search](https://docs.microsoft.com/azure/search/search-what-is-azure-search): the client application provides the search user interface.
+* A server A Function app that calls the Azure.Search.Documents query a search index.: It leverages the [Azure SDK for Javascript/Typescript for Cognitive Search](https://www.npmjs.com/package/@azure/search-documents).
```
Comment (HeidiSteen) [inline README.md:22]: It creates an index as well. After you run this app, you will have a new search index on your service called good-books.

File: search-website/README.md [hunk 2]
```diff
@@ -19,44 +19,51 @@ To run the sample locally, follow the steps below.
 ### Prerequisites
 
 - A GitHub account
+- A Cognitive Search resource using the `Free` pricing tier. This can be created in VS Code with the [Cognitive Search extention](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azurecognitivesearch).  Copy following resources values before continuing:
+    - YOUR-SEARCH-RESOURCE-NAME
+    - YOUR-SEARCH-ADMIN-KEY - used only in bulk import for inserting
```
Comment (HeidiSteen) [inline search-website/README.md:24]: Admin key is also used for creating an index

File: search-website/README.md [hunk 3]
```diff
@@ -19,44 +19,51 @@ To run the sample locally, follow the steps below.
...
+    git clone https://github.com/Azure-Samples/azure-search-javascript-samples
     ```
 
-## Run the app locally
+1. Open that local directory in Visual Studio Code.
+
+### Run bulk import to the Search Index
+
+1. Open the `./search-website/bulk-insert/bulk_insert_books.js`.
+1. Download the [books.csv](https://raw.githubusercontent.com/zygmuntz/goodbooks-10k/master/books.csv) to the same directory as the `bulk_insert_books.js` file.
```
Comment (HeidiSteen) [inline search-website/README.md:44]: when I cloned this repo, I got a _local_ books.csv file -- I don't think I have to download anything.  I downloaded the code from diberry so maybe the final version won't have this .csv file??

Comment (diberry) [reply search-website/README.md:44]: That's a mistake - I need to remove that file.

File: search-website/README.md [hunk 4]
```diff
@@ -19,44 +19,51 @@ To run the sample locally, follow the steps below.
...
+
+    * YOUR-SEARCH-RESOURCE-NAME
+    * YOUR-SEARCH-ADMIN-KEY
+
+1. Open a terminal in VS Code from this directory by selecting the directory, then selecting **Open Integrated Terminal**.
+ 
+1. Run the bulk import script:
+
+    ```bash
+    node bulk_insert_books.js
```
Comment (HeidiSteen) [inline search-website/README.md:55]: I get this error:
Windows PowerShell
Copyright (C) Microsoft Corporation. All rights reserved.

Try the new cross-platform PowerShell https://aka.ms/pscore6

PS C:\Users\heidist\azure-search-javascript-samples\search-website\bulk-insert> node bulk_insert_books.js
internal/modules/cjs/loader.js:883
  throw err;
  ^

Error: Cannot find module 'csv-parser'
Require stack:
- C:\Users\heidist\azure-search-javascript-samples\search-website\bulk-insert\bulk_insert_books.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:880:15)
    at Function.Module._load (internal/modules/cjs/loader.js:725:27)
    at Module.require (internal/modules/cjs/loader.js:952:19)
    at require (internal/modules/cjs/helpers.js:88:18)
    at Object.<anonymous> (C:\Users\heidist\azure-search-javascript-samples\search-website\bulk-insert\bulk_insert_books.js:8:15)
    at Module._compile (internal/modules/cjs/loader.js:1063:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
    at Module.load (internal/modules/cjs/loader.js:928:32)
    at Function.Module._load (internal/modules/cjs/loader.js:769:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:72:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    'C:\\Users\\heidist\\azure-search-javascript-samples\\search-website\\bulk-insert\\bulk_insert_books.js'
  ]
}
PS C:\Users\heidist\azure-search-javascript-samples\search-website\bulk-insert>

File: search-website/README.md (hunk 3 above)
Comment (HeidiSteen) [reply search-website/README.md:44]: Is there a reason why we can't provide the .csv?  It stabilizes the experience.  If someone changes the dataset on the public site, our index or queries could break.

**Comment by diberry** [comment]
@HeidiSteen We don't own the file - it is open source so guidelines are to leave it where it is unless we intend to change it.

File: search-website/README.md [hunk 5]
```diff
@@ -19,44 +19,57 @@ To run the sample locally, follow the steps below.
...
+1. Open the `./search-website/bulk-insert/bulk_insert_books.js`.
+1. Download the [books.csv](https://raw.githubusercontent.com/zygmuntz/goodbooks-10k/master/books.csv) to the same directory as the `bulk_insert_books.js` file.
+1. Replace the following variables in the `bulk_insert_books.js` with your own values to authenticate with the Azure Search SDK:
 
-### Running the front-end
+    * YOUR-SEARCH-RESOURCE-NAME
+    * YOUR-SEARCH-ADMIN-KEY
 
-1. From a terminal in the `./search-website` root, install front-end dependencies...
+1. Open a terminal in VS Code from this directory by selecting the directory, then selecting **Open Integrated Terminal**.
```
Comment (HeidiSteen) [inline search-website/README.md:50]: The option "Open in Integrated Terminal" is only available on folders, so I did this:  PS C:\Users\heidist\azure-search-javascript-samples\search-website\bulk-insert> npm install   

Obviously it's installing a boatload of software in that folder, which is kind of crazy. I have node_modules elsewhere, under %app-data%.   I'm going to blow this away and start over.

Comment (diberry) [reply search-website/README.md:50]: So node-modules in app-data is globally installed and not local to the project. Most JS development is done local to the project and only CLIs or computer-wide tooling is installed globally. This project format of package.json in the exact project folder/subfolder is done on purpose and is common with JS repos that hold more than 1 project .

**Comment by diberry** [comment]
@dereklegenzoff  @HeidiSteen can you please merge?

**Pull request merged**